goty-server/src/main/kotlin/com/aleinin/goty/submit/Submission.kt

```diff
@@ -0,0 +1,14 @@
+package com.aleinin.goty.submit
+
+import java.util.UUID
+
+data class Submission(
```

Comment:
To build on my other comment (read that first if you haven't) 

To me what makes the most sense (and i'm open to discussion of course) 

data class Submission(
  val id: UUID,
  val name: String,
  val gamesOfTheYear: List<RankedGame>
  val mostAnticipated: Game?,
  val bestOldGame: Game?
  enteredGiveaway: Boolean,
  val enteredOn: Long,
  val updatedOn: Long
)